src/Core/Crawl/groupobject.cpp

```diff
@@ -22,10 +22,20 @@ void GroupObject::render(unsigned int tbfMsec) {
     for (ClasterItem* object: objects()) {
 
         if (Localpropertys *props = getLocalpropertys(object->guiId())) {
-            if (!props->_rotation.isNull())
```

Comment:
you must to restore old code because you check rotation of child object but try change rotation for main object.

1. you need to check child objects rotation. if a rotation is not null then update a child's object rotation.
2. and anyway you need to update a position for main object using a rotation property.